test/blocks/tabs/tabs.test.js

```diff
@@ -35,4 +42,58 @@ describe('tabs', () => {
     await sendKeys({ press: 'Space' });
     expect(unSelectedBtn2.ariaSelected).to.equal('false');
   });
+
+  it('paddles are disabled when tabList is not scrollable', async () => {
+    setViewport({ width: DESKTOP_WIDTH, height: HEIGHT });
+    window.dispatchEvent(new Event('resize'));
+    await delay(200);
```

Comment:
Could some `useFakeTimers` variation be used instead of actually waiting for a set time? A lot of small delays can lead to long run times for unit tests

Reply:
Unfortunately `useFakeTimers` doesn't work here.